04_removeFromArray/removeFromArray.js

```diff
@@ -1,6 +1,26 @@
-const removeFromArray = function() {
+//This line declares function that can accept multiple arguments and puts them in an array
+const removeFromArray = function (...values) {
```

Comment:
This solution works but a bit by chance 😅 

Let's say you call the function like this:
```js
removeFromArray([1, 2, 3, 4], 1, 4)
```
Then you take the array part as you know this is the first in the arguments (which is good):
```js
const array = values[0] // [1, 2, 3, 4]
```
When `filter` is called on this array the comparison inside has one flaw, values are not only the rest of the arguments after the array but also the actual itself
```js
values.includes(item)
// the same as
[[1, 2, 3, 4], 1, 4].includes(item)
```
When `filter` is iterating through the array what you get is:
```js
[[1, 2, 3, 4], 1, 4].includes(1)
// includes is going through he array and checks if anything equals 1
[1, 2, 3, 4] === 1 // false
1 === 1 // true
4 === 1 // false
```
You should already see that the first comparison where `number` is compared with `array` will never be equal, and they shouldn't even be compared at all.

The most simple solution would be to extract the first argument and then the rest. Here is even a very similar example of how this can be done, you just have to modify it a bit: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters#syntax